jsonschema-core.xml

```diff
@@ -2757,9 +2756,9 @@
                     Implementations MAY elect to provide additional information.
                 </t>
 
-                <section title="Keyword Relative Location">
+                <section title="Subschema Relative Location">
                     <t>
-                        The relative location of the validating keyword that follows the path
+                        The relative location of the validating subschema that follows the path
```

Comment:
I kind of want to take "relative" out of this altogether, particularly since we are dropping "absolute" from `schemaLocation`.  They only really made sense in contrast to each other.